class_person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, parent_permission: true, name: 'Unknow')
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? or @parent_permission == true
```

Comment:
- [ ] Once more kindly check the list of [Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md) requirements. The `and` and `or` keywords are banned. Always use `&&` and `||` instead